tests/example_test.go

```diff
@@ -129,7 +129,7 @@ func TestExample(t *testing.T) {
 		{
 			example: "https://github.com/terraform-aws-modules/terraform-aws-vpc",
 			// TODO[pulumi/pulumi#13743]: unknown property 'domain' among [...]
-			skip: allLanguages,
+			skip: newStringSet(csharp, golang, python, typescript),
```

Comment:
This is still `allLanguages`.  Why did we change this, `allLanguages` seems more expressive to me?

But really what alerted me to look at this change is that we should probably add a test for the behaviour of  `TemplateJoinExpr`.